### DISCUSSION TITLE
Lay-Z-Spa app heater toggle switch question

### DISCUSSION THREAD

**Discussion by WesWesthaver** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#609] [Q&A]
The "Heater" toggle switch when On sometimes shows a Red dot and other times a Green dot. Am I correct in interpreting the toggle as indicating more than simply On/Off?

[Heater Switch Image](http://www.westhaver.com/heater_switch.png)

Should I interpret the On/Off toggle as Heater Enabled/Disabled.

And when the toggle is On:

- Red dot means the heater is enabled and is currently running?
- Green dot means the heater is enabled but not currently running?

-Wes

**Comment by visualapproach** [comment]
Correct
